portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/SharedComponents/PoliciesExpansion.tsx

```diff
@@ -0,0 +1,249 @@
+            const requestFlow = apiPolicies.request;
+            for (const requestFlowAttachedPolicy of requestFlow) {
+                const { policyId, policyName, policyVersion, uuid } =
+                    requestFlowAttachedPolicy;
+                if (policyId === null) {
+                    /**
+                     * Handling migration flow.
+                     */
+                    requestFlowList.push({
+                        ...defaultPolicyForMigration,
+                        name: policyName,
+                        displayName: policyName,
+                        applicableFlows: ['request'],
+                        uniqueKey: uuid,
+                    });
+                } else {
+                    const policyObj = allPolicies?.find(
+                        (policy: PolicySpec) => 
+                            policy.name === policyName && 
+                            policy.version === policyVersion,
+                    );
+                    if (policyObj) {
+                        requestFlowList.push({ ...policyObj, uniqueKey: uuid });
+                    } else {
+                        // console.error("Cannot find policyObj for policyId: " + policyId);
+                        Alert.error(intl.formatMessage({
+                            id: 'Cannot.Find.PolicyObj.For.PolicyId',
+                            defaultMessage: 'Cannot find policyObj for policyId: ',
+                        }) + policyId);
+                    }
+                }
+            }
+            setRequestFlowPolicyList(requestFlowList);
+
+            /**
+             * Populate response flow attached policy list.
+             */
+            const responseFlowList: AttachedPolicy[] = [];
+            const responseFlow = apiPolicies.response;
+            for (const responseFlowAttachedPolicy of responseFlow) {
+                const { policyId, policyName, policyVersion, uuid } =
+                    responseFlowAttachedPolicy;
+                if (policyId === null) {
+                    /**
+                     * Handling migration flow.
+                     */
+                    responseFlowList.push({
+                        ...defaultPolicyForMigration,
+                        name: policyName,
+                        displayName: policyName,
+                        applicableFlows: ['response'],
+                        uniqueKey: uuid,
+                    });
+                } else {
+                    const policyObj = allPolicies?.find(
+                        (policy: PolicySpec) => 
+                            policy.name === policyName && 
+                            policy.version === policyVersion,
+                    );
+                    if (policyObj) {
+                        responseFlowList.push({ ...policyObj, uniqueKey: uuid });
+                    } else {
+                        // console.error("Cannot find policyObj for policyId: " + policyId);
```

Comment:
```suggestion
                        
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)